app/assets/javascripts/app/helpers/handlebars-helpers.js

```diff
@@ -78,9 +75,9 @@ Handlebars.registerHelper('personImage', function(person, size, imageClass) {
   size = ( !_.isString(size) ) ? "small" : size;
   imageClass = ( !_.isString(imageClass) ) ? size : imageClass;
 
-  return _.template('<img src="<%= src %>" class="<%= img_class %>" title="<%= title %>" alt="<%= title %>" />')({
+  return _.template("<img src=\"<%= src %>\" class=\"<%= imageClass %>\" title=\"<%= title %>\" alt=\"<%= title %>\" />")({
```

Comment:
Uh it did complain about single quotes here? @svbergerem shouldn't single quotes be okay if double quotes are part of the string?